Pack3/games/AwShirt/content/AwShirtSlogans.jet

```diff
@@ -1052,7 +1052,7 @@
         },
         {
             "id": 40555,
-            "suggestion": "Sero'ha fwa tskxekeng fpi tur slu",
+            "suggestion": "Sero'ha fwa tskxekeng fte tur slu",
```

Comment:
With fte instead of fpi it would be "txur slivu", and I didn't notice there was a missing "si" before
But now I think the whole thing could be simplified into "So'yu tskxekengä letokx"